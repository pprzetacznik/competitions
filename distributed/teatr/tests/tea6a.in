100000000 1 99 -1 1000000 11677989741250451971
