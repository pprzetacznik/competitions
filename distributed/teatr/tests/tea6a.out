33251702881515
